src/datasource.ts

```diff
@@ -93,7 +90,7 @@ export class DataFrameDataSource extends DataSourceApi<DataframeQuery> {
 
   async testDatasource(): Promise<TestingStatus> {
     return lastValueFrom(
-      this.fetch<TableMetadataList>('GET', 'tables').pipe(
+      this.fetch<TableMetadataList>('GET', 'tables', { params: { take: 1 } }).pipe(
```

Comment:
This 'health' check only gets called when manually configuring the data source in Grafana (so really only matters for local development right now), but it was still bad to list the max amount of tables instead of getting just one.